options/locale/locale_en-US.ini

```diff
@@ -61,103 +61,103 @@ cancel = Cancel
-mailer_user = Sender User
-mailer_password = Sender Password
-register_confirm = Enable Register Confirmation
-mail_notify = Enable Mail Notifications
-server_service_title = Server and Other Services Settings
-offline_mode = Enable Offline Mode
-offline_mode_popup = Disable CDN so all resource files will be served locally.
-disable_gravatar = Disable Gravatar Service
-disable_gravatar_popup = Disable Gravatar and custom sources. All avatars must be uploaded by users or the default avatar will be used.
-federated_avatar_lookup = Enable Federated Avatars Lookup
+smtp_from = Send Email As
+smtp_from_helper = Email address Gitea will use. Enter a plain email address or use the "Name" <email@example.com> format.
+mailer_user = SMTP Username
+mailer_password = SMTP Password
+register_confirm = Require Email Confirmation to Register
+mail_notify = Enable Email Notifications
+server_service_title = Server and Third-Party Service Settings
+offline_mode = Enable Local Mode
+offline_mode_popup = Disable third-party content delivery networks and serve all resources locally.
+disable_gravatar = Disable Gravatar
+disable_gravatar_popup = Disable Gravatar and third-party avatar sources. A default avatar will be used unless a user locally uploads an avatar.
+federated_avatar_lookup = Enable Federated Avatars
 federated_avatar_lookup_popup = Enable federated avatar lookup using Libravatar.
-disable_registration = Disable Self-registration
-disable_registration_popup = "Disable self-registration; only admins will be able to create accounts."
+disable_registration = Disable Self-Registration
+disable_registration_popup = Disable user self-registration. Only administrators will be able to create new user accounts.
 openid_signin = Enable OpenID Sign-In
-openid_signin_popup = Enable user login via OpenID
-openid_signup = Enable OpenID Self-registration
-openid_signup_popup = Enable OpenID based Self-registration
-enable_captcha = Enable Captcha
+openid_signin_popup = Enable user sign-in via OpenID.
+openid_signup = Enable OpenID Self-Registration
+openid_signup_popup = Enable OpenID-based user self-registration.
+enable_captcha = Enable CAPTCHA
 enable_captcha_popup = Require a CAPTCHA for user self-registration.
-require_sign_in_view = Enable Require Sign In to View Pages
-require_sign_in_view_popup = "Only signed in users can view pages; visitors will only be able to see the sign in and up pages."
-admin_setting_desc = You do not need to create an admin account right now. The first user who registers on the site will gain admin access automatically.
-admin_title = Admin Account Settings
+require_sign_in_view = Require Sign-In to View Pages
+require_sign_in_view_popup = Limit page access to signed-in users. Visitors will only see the 'sign in' and registration pages.
+admin_setting_desc = Creating an administrator account is optional. The first registered user will automatically become an administrator.
```

Comment:
Emphasis on 'now' I think is important. Perhaps 'here' or 'on this page'?

Reply:
I think this wording is entirely self-contained. It now also fits in one line.

Reply:
@thehowl can you close or accept this if you agree to keep the review page as clean as possible? Thanks!